ably-ios/ARTAuth.m

```diff
@@ -99,6 +114,8 @@ - (instancetype)init {
         _authToken = nil;
```

Comment:
This should be `token` not `authToken`.  Also, we support `key` as an attribute for AuthOptions, but optionally support `keyName` and `keySecret` but this is not documented. See https://github.com/ably/ably-java/blob/master/src/io/ably/rest/Auth.java#L44-L129.  We should no longer refer to keyId and keyValue anywhere but for clarity instead refer to them as keyName and keySecret for consistency and to reduce ambiguity.  A key is then the complete `keyName:keySecret` and a `token` attribute is always the `token` string itself.